src/dialogs/more-info/controls/more-info-climate.ts

```diff
@@ -166,10 +166,7 @@ class MoreInfoClimate extends LitElement {
       </div>
       <ha-more-info-control-select-container>
         <ha-control-select-menu
-          .label=${this.hass.formatEntityAttributeName(
-            this.stateObj,
-            "hvac_mode"
-          )}
+          .label=${this.hass.localize("ui.card.climate.mode")}
```

Comment:
We should move this key somewhere else in the future, as this is not a `card`, but fine for now